impeller/entity/contents/atlas_contents.cc

```diff
@@ -235,13 +235,14 @@ bool AtlasContents::Render(const ContentContext& renderer,
       }
     }
 
-    Command cmd;
-    DEBUG_COMMAND_INFO(
-        cmd, SPrintF("DrawAtlas Blend (%s)", BlendModeToString(blend_mode_)));
-    cmd.BindVertices(vtx_builder.CreateVertexBuffer(host_buffer));
-    cmd.stencil_reference = entity.GetClipDepth();
-    auto options = OptionsFromPass(pass);
-    cmd.pipeline = renderer.GetPorterDuffBlendPipeline(options);
+#ifdef IMPELLER_DEBUG
```

Comment:
I only added ifdefs where we do runtime string creation.

Reply:
I prefer the macro style we had previously.  Did you find the string creation wasn't getting stripped out?

FWIW this could be transformed to a static string too since we only have so many blend modes.  I did that at some point but the PR got caught up and dropped.

Reply:
Strings were still getting formatted, then dropped. We could add more static strings for these, I wouldn't be opposed. Feedback I got on the macro approach was that everyone hated it 😆